### PR TITLE
Yse/tc 03.03.004 verify freestyle project created

### PR DESCRIPTION
**Pull request by Yuriy-SE**
TC_03.03.004 | Create Freestyle Project > Configure new Freestyle project> Verify that Freestyle Project is created and verify elements on /configure page
https://trello.com/c/Xwy6IiKb/223-tc0303004-create-freestyle-project-configure-new-freestyle-project-verify-that-freestyle-project-is-created-and-verify-elements

### CODE REVIEW

**Review by tatvalverde — APPROVED** [review]

**Comment by qa-tester22** [comment]
TC_03.04.008  Create Pipeline project  Not unique name  Message invalid item name (failed).png    
     Такое имя уже существует. нужно изменить.

**Comment by Rom-T** [comment]
> TC_03.04.008 Create Pipeline project Not unique name Message invalid item name (failed).png Такое имя уже существует. нужно изменить.

Так этот TC относится к Create Freestyle Project

File: cypress/e2e/createFreestyleProject.cy.js [hunk 1]
```diff
@@ -24,4 +25,25 @@ describe('createFreestyleProject', () => {
 
         cy.get('textarea[name="description"]').should('exist').and('be.visible');
     });
+
+    it('TC_03.03.004 | Verify that Freestyle Project is created and verify elements on /configure page', function () {
+        const testCase = 'TC_03.03.004';
```
Comment (Rom-T) [inline cypress/e2e/createFreestyleProject.cy.js:30]: It is better to store the date in fixtures instead of creating a variable.

Comment (Yuriy-SE) [reply cypress/e2e/createFreestyleProject.cy.js:30]: may be. but I created this variable only for this test.

Comment (Yuriy-SE) [reply cypress/e2e/createFreestyleProject.cy.js]: ok, I moved it into the fixtures

**Comment by Yuriy-SE** [comment]
[TC_03.03.004 | Create Freestyle Project > Configure new Freestyle project> Verify that Freestyle Project is created and verify elements on /configure page](https://trello.com/c/Xwy6IiKb/223-tc0303004-create-freestyle-project-configure-new-freestyle-project-verify-that-freestyle-project-is-created-and-verify-elements)

**Review by Rom-T — APPROVED** [review]

**Pull request merged**